docs/reference/configuration.mdx

```diff
@@ -306,6 +306,8 @@ rate limiter. This could be combined with a second instance of
 
 ## `ipv6Subnet`
 
+> Added in `8.0.0` (/56 default), `7.6.0` (/64 default).
+
```

Comment:
We should probably add this for all of the options.

Reply:
Well, I mainly meant that we should document what version each option was added in. But, for ones where the default changed, then maybe we should do it like this.